### PR TITLE
[Docathon] Fix NO.22-NO.26 API label

### PR DESCRIPTION
**Pull request by zade23**
### PR types

Others

### PR changes

Docs 

### Description

#57363

| 序号 | Origin label name                       | Fixed  label name                             |
| ---- | --------------------------------------- | --------------------------------------------- |
| 22   | `api_base_layers_DynamicRNN`            | `api_paddle_nn_RNN`                           |
| 23   | `api_base_layers_conv2d`                | `api_paddle_nn_Conv2D`                        |
| 24   | `api_base_layers_create_global_var`     | `api_paddle_static_create_global_var`         |
| 25   | `api_base_layers_create_tensor`         | `api_paddle_Tensor_create_tensor`             |
| 26   | `api_base_layers_while_loop`            | `api_paddle_static_nn_while_loop`             |

### CODE REVIEW

File: python/paddle/static/nn/sequence_lod.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def sequence_conv(
     r"""
 
     Note:
-        Only receives Tensor as input. If your input is Tensor, please use conv2d Op.(base.layers.** :ref:`api_base_layers_conv2d` ).
+        Only receives Tensor as input. If your input is Tensor, please use conv2d Op.(base.layers.** :ref:`api_paddle_nn_Conv2D` ).
```
Comment (ooooo-create) [inline python/paddle/static/nn/sequence_lod.py]:
```suggestion
        Only receives Tensor as input. If your input is Tensor, please use conv2d Op.(base.layers.** :ref:`api_paddle_nn_functional_conv2d` ).
```

File: test/legacy_test/nets.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def simple_img_conv_pool(
     r"""
         :api_attr: Static Graph
 
-    The simple_img_conv_pool api is composed of :ref:`api_base_layers_conv2d` and :ref:`api_base_layers_pool2d` .
+    The simple_img_conv_pool api is composed of :ref:`api_paddle_nn_Conv2D` and :ref:`api_base_layers_pool2d` .
```
Comment (ooooo-create) [inline test/legacy_test/nets.py]:
```suggestion
    The simple_img_conv_pool api is composed of :ref:`api_paddle_nn_functional_conv2d` , :ref:`api_paddle_nn_functional_avg_pool2d` and :ref:`api_paddle_nn_functional_max_pool2d` .
```

**Review by ooooo-create — COMMENTED** [review]
image
从飞桨框架2.0版本开始，飞桨默认为开启了动态图开发模式
`api_base_layers_conv2d` -> `api_paddle_nn_functional_conv2d`

File: test/legacy_test/nets.py (hunk 2 above)
Comment (ooooo-create) [reply test/legacy_test/nets.py]: `api_paddle_nn_functional_conv2d` 后面少了个逗号

Comment (zade23) [reply test/legacy_test/nets.py]: ~~这条49行的后面新添加的 `api_paddle_nn_functional_avg_pool2d`  和 `api_paddle_nn_functional_max_pool2d`是出自哪里呢……~~
没事了，我在1.8和2.0映射文档中看到了 0.o

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**